test/unit/presenters/api/integrations_show_presenter_test.rb

```diff
@@ -50,8 +50,6 @@ def test_environments_have_same_config?
   end
 
   test 'state_modifier' do
-    @proxy.account.stubs(:provider_can_use?).with(:api_as_product).returns(true)
```

Comment:
I recommend not doing this change here to avoid conflicts with https://github.com/3scale/porta/pull/2291, where you actually remove this entire test.
But it's up to you 👍🏼